src/recipe/recipe.controller.ts

```diff
@@ -64,19 +67,50 @@ export class RecipeController {
   })
   @Get(':id')
   async fetchRecipe(
-    @Param('id', ParseIntPipe) id: number,
+    @Request() userIdObject: { user: { id: number } },
+    @Param('id', ParseIntPipe) recipeId: number,
   ): Promise<FetchRecipeResponse> {
-    const fetchedRecipe = await this.recipeService.fetchRecipe(id);
+    const fetchedRecipe = await this.recipeService.fetchRecipe(
+      recipeId,
+      userIdObject.user.id,
+    );
+
     return FetchRecipeResponse.from(fetchedRecipe);
   }
 
+  @HttpCode(200)
+  @UseGuards(JwtAuthGuard)
+  @ApiOperation({ summary: 'Get all users recipes' })
+  @ApiBearerAuth()
+  @ApiUnauthorizedResponse({ description: 'User is not authenticated' })
+  @ApiParam({
+    name: 'email',
+    description: 'Email of a user',
+  })
+  @Get('/user/:email')
+  async fetchUsersRecipes(
```

Comment:
`fetchUsersRecipes` is a bit confusing in my opinion. 
how about 
`findRecipesByAuthorEmail()` or ...OwnerEmail

Reply:
I think I'm going to stick with this fetch for now. I used `fetch` everywhere in project, so it would require lots of changes (mocks, tests, service, controller, dtos, and likely few things more).  I'll probably create a different refactor branch and do this in it.